src/Libraries/PythonNodeModels/Properties/AssemblyInfo.cs

```diff
@@ -12,4 +12,5 @@
 [assembly: Guid("a0debf47-0cf2-46e8-90b9-0c3d9b808fd7")]
 // Remove this when PythonEngineSelector become public
 [assembly: InternalsVisibleTo("DynamoPythonTests")]
-[assembly: InternalsVisibleTo("DynamoCoreWpfTests")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("DynamoCoreWpfTests")]
+[assembly: InternalsVisibleTo("IronPythonExtension")]
```

Comment:
I don't think this helps in the cause of externalizing the extension. We should use an event and a proper API.

Reply:
`RequestUpdateLoadedPythonEngines`?

Reply:
For this Core would need to be aware of the event on the extension, in order to register its scan method with it, which does not seem right. An extension is and should be aware of Core, not the other way round.